frontend/src/app/testResults/viewResults/DownloadResultsCsvModal.tsx

```diff
@@ -59,22 +57,17 @@ export const DownloadResultsCsvModal = ({
   };
 
   const [downloadTestResultsQuery, { loading }] =
-    useGetFacilityResultsForCsvWithCountLazyQuery({
+    useGetResultsForDownloadLazyQuery({
       variables,
       fetchPolicy: "no-cache",
-      onCompleted: (data: GetFacilityResultsForCsvWithCountQuery) =>
-        handleComplete(data),
+      onCompleted: (data: GetResultsForDownloadQuery) => handleComplete(data),
       onError: (error: ApolloError) => handleError(error),
     });
 
-  const handleComplete = (data: GetFacilityResultsForCsvWithCountQuery) => {
-    if (data?.testResultsPage?.content) {
+  const handleComplete = (data: GetResultsForDownloadQuery) => {
+    if (data?.resultsPage?.content) {
       try {
-        const csvResults = parseDataForCSV(
-          data.testResultsPage.content,
-          disabledFeatureDiseaseList,
-          filterParams
-        );
+        const csvResults = parseDataForCSV(data.resultsPage.content);
```

Comment:
So we will no longer be filtering diseases that we aren't supporting? 🤔

Reply:
Hmm so I think there was some desire for us to move filtering logic to the backend query instead of this being on the frontend, but now that I'm thinking about it, the results page frontend will probably always need to know the list of supported/unsupported diseases so it can populate the select list for Condition filter so I think we could keep using that `disabledFeatureDiseaseList` to handle it for the CSV and still be able to get rid of having to pass down `filterParams` to do other filtering on the CSV since that'll already be applied from the query response